src/directives/badge/badge.directive.ts

```diff
@@ -0,0 +1,276 @@
+import { AfterContentChecked, Directive, ElementRef, Input, OnChanges, OnDestroy, Renderer2, SimpleChanges } from '@angular/core';
+import { ColorService } from './../../services/color/color.service';
```

Comment:
Just noticed these import paths are starting with a `./` this can be removed as is isn't required.